offline-install/02.a_online_models.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+registry=registry:5000
+hf_token="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"
```

Comment:
Security: don't hard-code secrets, take them as arguments, e.g. as name of file containing the token.